Utils/trigger_contribution_build.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+import argparse
+import logging
+import os
+
+import urllib3
+from github import Github, GithubException
+from github.Issue import Issue
+from github.PaginatedList import PaginatedList
+from gitlab import Gitlab
+from gitlab.exceptions import GitlabError
+from gitlab.v4.objects.branches import ProjectBranch
+from gitlab.v4.objects.projects import Project
+from Tests.scripts.utils.log_util import install_logging
+
+urllib3.disable_warnings()
+
+DEFAULT_CI_PIPELINE_SOURCE = "contrib"
+GITLAB_DEFAULT_PROJECT_ID = 1061
```

Comment:
```suggestion
GITLAB_DEFAULT_PROJECT_ID = os.getenv('CI_PROJECT_ID') or 1061
```

Reply:
We dont want the constant to be define by the env vars, I think we should keep this a const..